client/css/style.css

```diff
@@ -493,6 +493,7 @@ kbd {
 #sidebar .network,
 #sidebar .network-placeholder {
 	margin-bottom: 30px;
+	touch-action: pan-y;
```

Comment:
Shouldn't this be on `.networks`, or `.sidebar` even?

Reply:
It may help to add this on `.networks`, it has no effect on `.sidebar` (it didn't when I was testing at least) but it's nice to have there to.